presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -660,7 +662,12 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         NATIVE_STATS_BASED_FILTER_REORDER_DISABLED,
                         "Native Execution only. Disable stats based filter reordering.",
                         false,
-                        true));
+                        true),
```

Comment:
Can the user modify this as a session property ? That shouldn't be allowed right ?

Reply:
Changed to hidden property, thanks for pointing it out.